<ui version="4.0">
 <class>PrintNomenclUi</class>
 <widget class="QDialog" name="PrintNomenclUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="setWidthLA">
     <property name="toolTip">
      <string>Define hanging indentation/label length for the nomenclature list.</string>
     </property>
     <property name="text">
      <string>&amp;List Indentation:</string>
     </property>
     <property name="buddy">
      <cstring>setWidthCO</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QComboBox" name="setWidthCO">
     <property name="toolTip">
      <string>Define hanging indentation/label length for the nomenclature list.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="valueLA">
     <property name="text">
      <string>Custom &amp;Width:</string>
     </property>
     <property name="buddy">
      <cstring>valueLE</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="valueLE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Custom value. &quot;List Indentation&quot; needs to be set to &quot;Custom&quot;.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="LengthCombo" name="unitLC"/>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>74</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyPB">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LengthCombo</class>
   <extends>QComboBox</extends>
   <header>LengthCombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>valueLE</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>applyPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
